src/web/app/pages-instructor/student-list/student-list-section-data.ts

```diff
@@ -16,5 +16,56 @@ export interface StudentListStudentData {
   email: string;
   status: string;
   team: string;
+  section?: StudentListSectionData;
```

Comment:
Why not just store `sectionName: String` here?

It is confusing to have a list of students in a section nested in every student.

Reply:
Did that to keep the these two properties which are needed by table: `isAllowedToViewStudentInSection: boolean` and `isAllowedToModifyStudent: boolean`. Should I make those properties of student too? 

Also, I set is as optional because there are other classes that are dependent on it.

Reply:
> Should I make those properties of student too?

I guess you should.

> I set is as optional because there are other classes that are dependent on it.

This is not a good reason why this field should be optional.

Reply:
Updated. What I did instead was to create a new interface to store the flattened data which has both the student and section properties.